tests/test_meshtastic_utils.py

```
@@ -1320,6 +1320,19 @@ def setUp(self):
+    def tearDown(self):
+        """Drain any coroutines submitted via run_coroutine_threadsafe."""
+        import asyncio
+
+        import mmrelay.meshtastic_utils as mu
+
+        loop = mu.event_loop
+        if loop and not loop.is_closed():
+            try:
+                loop.run_until_complete(asyncio.sleep(0))
+            except RuntimeError:
+                pass
```

Ruff (0.15.9)

[warning] Use `contextlib.suppress(RuntimeError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(RuntimeError): ...`

(SIM105)